src/index.jsx

```diff
@@ -0,0 +1,133 @@
+import React, {PureComponent} from 'react'
+
+import {create as createJSS, getDynamicStyles} from 'jss'
+import preset from 'jss-preset-default'
+
+import filterProps from './utils/filter-props'
+
+
+const JSS = createJSS(preset())
```

Comment:
yep, fix it here https://github.com/lttb/jss-styled/pull/1/commits/7eb58c1446121366ece33745bdf55a35cb887a2d#diff-73642d587163f2eb4d72f0fa6ebfba02R9